cwatch.php

```diff
@@ -459,16 +543,25 @@ public function tabClientActions($package, $service, array $get = null, array $p
         $client = $this->Clients->get($service->client_id, false);
         $email = $client->email;
         $this->loadApi($user, $pass, $sandbox);
-        if (!empty($post)) 
-        {
-            $sites = $this->api->addsite(['email' => $email, 'domain' => $post['domainname'], 'licenseKey' => $licensekey, 'initiateDns' => $post['initiateDns'] == 1 ? true : false, 'autoSsl' => $post['autoSsl'] == 1 ? true : false]);
-            if (!empty($sites->errorMsg)) {
-                $this->Input->setErrors(['api' => ['internal' => $sites->errorMsg]]);
+        $sitesdata = $this->api->getsites($email);
+        $usedsites=json_decode($sitesdata->resp,true);
+        if (!empty($post)){
+            $sitecount=0;
+            foreach($usedsites as $site){
+                if($site->licenseKey==$licensekey){
+                    $sitecount +=1;
+                }
+            }
+            if($sitecount<$service_fields->number_sites || $service_fields->number_sites==0){
```

Comment:
A few more cases here where there is no space between operators (e.g. `==`, `){`, etc.). Lines shouldn't be longer than 120 characters. You should be able to setup your IDE to show a vertical bar that you can use as a guide for the character limit.